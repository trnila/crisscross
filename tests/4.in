8 5
11  2222
 3333 3
44  55 8
 4  6 6
44444 8
     2