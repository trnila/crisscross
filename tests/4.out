┌─┐         ┌─┬─┐
│1│         │2│2│
└─┼─┬─┬─┬─┐ ├─┼─┘
  │3│3│3│3│ │3│
┌─┼─┼─┴─┼─┼─┼─┼─┐
│4│4│   │5│5│ │8│
└─┼─┤   ├─┼─┼─┼─┘
  │4│   │6│ │6│
┌─┼─┼─┬─┼─┤ ├─┤
│4│4│4│4│4│ │8│
└─┴─┴─┴─┴─┘ └─┘

